src/ElmFormat/Render/Box.hs

```diff
@@ -2120,6 +2072,106 @@ formatComment comment =
         AST.CommentTrickBlock c ->
             mustBreak $ row [ punc "{--", literal c, punc "-}" ]
 
+formatIfExpression ::
```

Comment:
I don't know where I should put my function in the file.
It seemed to me at first that some main top level functions may have been sorted alphabetically, and that's why I put it between `formatComment*` and `formatLiteral`, but there are a lot of exceptions that could make this guess completely false.

Reply:
sounds fine.  I usually just put it directly below the function where I first use it 🤷‍♂